bin/generate_repos.py

```diff
@@ -191,19 +193,38 @@ def create_or_update_repo(folder, subdirectory, which_branch):
 is_team = False
 if eg is not None:
     is_team = eg.eg_team_assignment
+    # gradeable is not a vcs gradeable
+    if eg.eg_vcs_host_type == -1:
+        print((WARNING + "Warning: Semester '{}' and Course '{}' contains gradeable_id '{}' but it is not a VCS gradeable." + DEFAULT).format(args.semester, args.course, args.repo_name))
+        print("exiting")
+        sys.exit()
+# gradeable does not exist in the course
 elif not args.non_interactive:
-    print ("Warning: Semester '{}' and Course '{}' does not contain gradeable_id '{}'.".format(args.semester, args.course, args.repo_name))
-    response = input ("Should we continue and make individual repositories named '"+args.repo_name+"' for each student? (y/n) ")
-    if not response.lower() == 'y':
+    print ((WARNING + "Warning: Semester '{}' and Course '{}' does not contain gradeable_id '{}'." + DEFAULT).format(args.semester, args.course, args.repo_name))
```

Comment:
Can you remove the colors on the print. This output may go to a log so it doesn't make sense to put the color there.

Reply:
Sure, done.